rocky/reports/report_types/open_ports_report/report.py

```diff
@@ -0,0 +1,53 @@
+from datetime import datetime
+from logging import getLogger
+from typing import Any, Dict
+
+from django.utils.translation import gettext_lazy as _
+
+from octopoes.models import Reference
+from octopoes.models.ooi.dns.zone import Hostname
+from octopoes.models.ooi.network import IPAddressV4, IPAddressV6
+from octopoes.models.path import Path
+from reports.report_types.definitions import Report
+
+logger = getLogger(__name__)
+
+
+class OpenPortsReport(Report):
+    id = "open-ports-report"
+    name = _("Open Ports Report")
+    description = _("Find open ports of IP addresses")
+    plugins = {"required": ["nmap"], "optional": ["shodan"]}
+    input_ooi_types = {Hostname, IPAddressV4, IPAddressV6}
```

Comment:
```suggestion
    input_ooi_types = {Hostname, IPAddress}
```

Reply:
The input OOIs that you can select in the report flow are `IPAddressV4` and `IPAddressV6`. `IPAddress` isn't an OOI type that is created and can be be selected, so this suggested change is not correct as far as I can see.